items/weapons.xml

```diff
@@ -1250,7 +1250,7 @@
   </CraftedItem>
   <CraftedItem id="crpg_aserai_noble_sword_3_t5" name="{=bohbzGjg}Decorated Scimitar with Wide Grip" crafting_template="crpg_OneHandedSword" is_merchandise="false" culture="Culture.aserai" modifier_group="sword">
     <Pieces>
-      <Piece id="crpg_aserai_noble_sword_3_t5_blade" Type="Blade" scale_factor="100" />
+      <Piece id="crpg_aserai_noble_sword_3_t5_blade" Type="Blade" scale_factor="101" />

```

Comment:
yes this is how you edit length. 100 is 100% of the piece length. So 200 will be double the piece length